main.go

```diff
@@ -107,6 +123,8 @@ func run(cfg *config) {
 
 	for _, tupeloContainer := range cfg.TupeloImages {
 		fmt.Printf("Running test suite with %v\n", tupeloContainer)
+		tupeloImage := strings.Split(tupeloContainer, " ")[0]
```

Comment:
Should we just go ahead and make the `TupeloImages` support the objects in the docker-compose like syntax: `image: `, `cmd: `, etc? Maybe rename `TupeloImages` to `TupeloEnvironments`?

Reply:
Yeah that would be much more robust. If you did that already on your branch or you otherwise beat me to it, I’ll gladly close this PR in favor of yours.